rest-api/test/runner.py

```diff
@@ -94,7 +94,19 @@ def main(sdk_path, test_path, test_pattern):
 
   args = parser.parse_args()
 
+  from coverage import Coverage
+  cov = Coverage(omit=[
+    '*/lib/*',
+    '*/appengine-mapreduce/*',
+    '*/site-packages/*',
+    '*/google_appengine/*',
+    '*/test/*',
+  ])
+  cov.start()
```

Comment:
How much does this slow down tests?

Thank you for adding this capability -- but if it slows down tests appreciably when we're not particularly paying attention to coverage, we should make it flagged off by default.

Reply:
It seems to add around 5-15 seconds.  Without it, it looks like the tests run in 55-65s, with, 65-80s.   Bear in mind though that I didn't rigorously profile this, I just ran with and without a few times and observed - also those numbers are only for the unit tests, they don't include the client tests (which are not affected).

Reply:
OK. Let's hide it behind a flag then, and add the flag to your instructions.

Reply:
Done.  The flag is `-c` and it is documented in `rest-api/README.md` with the rest of the testing instructions.